contributors/devel/conformance-tests.md

```diff
@@ -63,7 +63,7 @@ credentials.
 
 ```sh
 # build test binaries, ginkgo, and kubectl first:
-make WHAT=test/e2e/e2e.test && make WHAT=ginkgo && make WHAT=cmd/kubectl
```

Comment:
@benmoss i remember seeing another syntax, so a single make command will suffice. WDYT?

```
make WHAT="test/e2e/e2e.test  vendor/github.com/onsi/ginkgo/ginkgo cmd/kubectl"
```

Reply:
@dims perfect, I just updated the commit